planning/behavior_velocity_planner/include/scene_module/scene_module_interface.hpp

```diff
@@ -123,7 +123,7 @@ class SceneModuleManagerInterface
       tier4_planning_msgs::msg::StopReason stop_reason;
       scene_module->setPlannerData(planner_data_);
       scene_module->modifyPathVelocity(path, &stop_reason);
-      stop_reason_array.stop_reasons.emplace_back(stop_reason);
+      if (stop_reason.reason != "") stop_reason_array.stop_reasons.emplace_back(stop_reason);
```

Comment:
It's preferable to add braces.
```suggestion

      if (stop_reason.reason != "") {
          stop_reason_array.stop_reasons.emplace_back(stop_reason);
      }
```